package.json

```diff
@@ -19,6 +19,8 @@
     "build": "yarn workspaces run build",
     "dev:burnr": "yarn workspace @substrate/connect run build && yarn workspace @substrate/burnr run build && yarn workspace @substrate/burnr run start",
     "dev:smoldot-browser-demo": "yarn workspace @substrate/smoldot-browser-demo run dev",
+    "deploy:gh-pages:smoldot-browser-demo": "cd projects/smoldot-browser-demo && yarn deploy:gh-pages",
```

Comment:
I don't think its a good idea to have a "deploy everything everywhere" script in the root of the project.  I'll jot some thoughts down in the issue I created for deploying multiple apps #57

Reply:
I would just assume that we need to update all apps when we're introducing any significant changes to the `@substrate` packages, there are changes to the `@polkadot` packages, Smoldot or even Substrate, so instead of triggering the updates all manually , there would be one command to deploy all.

Reply:
Let's discuss in #57.